2 - RTB/Documentazione esterna/Verbali/23-12-15/23-12-15.typ

```diff
@@ -0,0 +1,63 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "15/12/23",
+  subTitle: "",
+  docType: "verbale",
+  authors: (
+    "Nardo Silvio",
+  ),
+  externalParticipants : (
+    (name: "Alberton Michela", role: "Membro azienda proponente"),
+    (name: "Bassani Matteo", role: "Referente aziendale"),
+    (name: "Ruffin Filippo", role: "Membro azienda proponente"),
+  ), 
+  timeStart: "12:00",
+  timeEnd: "12:30",
+);
+
+= Ordine del giorno
+Sono stati esposti i progressi sui PoC rispetto al precedente meeting esterno e sono sorte discussioni relativamente a:
+- Analisi dei Requisiti;
+- tecnologie da utilizzare e utilizzate;
+- feature dei PoC;
+- meeting futuri.
+
+== Considerazioni sull'Analisi dei Requisiti
+Sono stati discussi i requisiti non funzionali identificati.
+Non sono stati identificati particolari vincoli sulle performance.
+
+=== Vincoli sui browser 
+In seguito all'utilizzo delle tecnologiche da noi individuate, in particolar modo per la libreria Three.js, è necessario che i browser interessati dispongano di una versione minima atta a supportare tale tecnologia mediante  la componente grafica WebGL. Tale osservazione ha trovato conferma da parte del proponente, il quale ha approvato l'implementazione dell'applicativo per i principali browswer:
```

Comment:
- ~In seguito... utilizzo~ Per adottare le tecnologie da noi individuate
- I browser non dispongono di una versione minima, siamo noi a definirle
- Doppio spazio dopo "mediante"